src/SDKs/SqlManagement/Management.Sql/Generated/Models/DataMaskingRule.cs

```diff
@@ -0,0 +1,230 @@
+        /// the character to use for masking the unexposed part of the string.
+        /// Otherwise, this parameter will be ignored.</param>
+        /// <param name="location">The location of the data masking
+        /// rule.</param>
+        /// <param name="kind">The kind of Data Masking Rule. Metadata, used
+        /// for Azure portal.</param>
+        public DataMaskingRule(string schemaName, string tableName, string columnName, DataMaskingFunction maskingFunction, string id = default(string), string name = default(string), string type = default(string), string dataMaskingRuleId = default(string), string aliasName = default(string), DataMaskingRuleState? ruleState = default(DataMaskingRuleState?), string numberFrom = default(string), string numberTo = default(string), string prefixSize = default(string), string suffixSize = default(string), string replacementString = default(string), string location = default(string), string kind = default(string))
+            : base(id, name, type)
+        {
+            DataMaskingRuleId = dataMaskingRuleId;
+            AliasName = aliasName;
+            RuleState = ruleState;
+            SchemaName = schemaName;
+            TableName = tableName;
+            ColumnName = columnName;
+            MaskingFunction = maskingFunction;
+            NumberFrom = numberFrom;
+            NumberTo = numberTo;
+            PrefixSize = prefixSize;
+            SuffixSize = suffixSize;
+            ReplacementString = replacementString;
+            Location = location;
+            Kind = kind;
+            CustomInit();
+        }
+
+        /// <summary>
+        /// An initialization method that performs custom operations like setting defaults
+        /// </summary>
+        partial void CustomInit();
+
+        /// <summary>
+        /// Gets the rule Id.
+        /// </summary>
+        [JsonProperty(PropertyName = "properties.id")]
+        public string DataMaskingRuleId { get; private set; }
+
+        /// <summary>
+        /// Gets or sets the alias name. This is a legacy parameter and is no
+        /// longer used.
+        /// </summary>
+        [JsonProperty(PropertyName = "properties.aliasName")]
+        public string AliasName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the rule state. Used to delete a rule. To delete an
+        /// existing rule, specify the schemaName, tableName, columnName,
+        /// maskingFunction, and specify ruleState as disabled. However, if the
+        /// rule doesn't already exist, the rule will be created with ruleState
+        /// set to enabled, regardless of the provided value of ruleState.
+        /// Possible values include: 'Disabled', 'Enabled'
+        /// </summary>
+        [JsonProperty(PropertyName = "properties.ruleState")]
+        public DataMaskingRuleState? RuleState { get; set; }
+
+        /// <summary>
+        /// Gets or sets the schema name on which the data masking rule is
```

Comment:
This is generated code and changing the formatting would just be overwritten the next time someone generates client